src/3id/keyring.js

```diff
@@ -60,7 +58,7 @@ class Keyring {
   }
 
   getDBKey () {
-    return ec.keyFromPrivate(this.signingKey.privateKey.slice(2))
+    return this.signingKey.privateKey.slice(2)
```

Comment:
is this supposed to be the priv key or pub key?

Reply:
it's supposed to be the priv key since it's passed to orbitdb for writing new entries.

Reply:
ok, yeah didn't read through rest of code not in diff, just wondering what ec.keyFromPrivate was doing before then

Reply:
The new lines which do `importFromPrivate` basically calls the same function, but with the correct elliptic instance.

Reply:
ok yeah, and you could have put this line here as well to the same effect `await this._orbitdb.keystore.importPrivateKey(dbKey)`

Reply:
Well yeah, if we pass orbitdb I guess. Or pass the keystore instance?

Reply:
oh I see, yeah I think is fine 👍

Reply:
that this impl is fine or that we should pass the keystore?

Reply:
this implementation

Reply:
ok great  😅